### ISSUE TITLE
Luxon localizer has problem displaying Month in Chinese language

### ISSUE THREAD

**Issue by RemLawrence** [jquense/react-big-calendar#2185]
### Clear and concise description of the problem

When choosing Chinese/Japanese for calendar, in the header, the luxon localizer will only display the number of the months (e.g. 6 2017), instead of the actual localized month (e.g. 六月 2017)
Read the issue from luxon side: https://github.com/moment/luxon/issues/628

### Suggested solution

[This line](https://github.com/jquense/react-big-calendar/blob/master/src/localizers/luxon.js#L37) should be `LLLL yyyy` instead.

### Validations

- [X] Read the [docs](http://jquense.github.io/react-big-calendar/examples/index.html).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### Would you like to open a PR for this feature?

- [X] I'm willing to open a PR

**Comment by cutterbl** [comment]
As a temporary fix, you can apply your own [monthHeaderFormat](http://jquense.github.io/react-big-calendar/examples/index.html?path=/docs/props--formats-month-header-format)

**Comment by RemLawrence** [comment]
Yes, already did that. Just want to point it out for future reference. Thanks!
